src/components/Menu/Menu.tsx

```diff
@@ -1,36 +1,79 @@
+// Menu.tsx
+import { useEffect, useState } from 'react';
 import profileIcon from '../../assets/Menu icons/profile_icon.jpg';
 import orderIcon from '../../assets/Menu icons/order_icon.jpg';
 import couponIcon from '../../assets/Menu icons/coupon_icon.jpg';
 import discountIcon from '../../assets/Menu icons/discount_icon.jpg';
-export const Menu = () => {
+import UnauthenticatedProfile from '../../pages/UnauthenticatedProfile';
+import Profile from '../../pages/Profile';
+import { getUserData } from '../../services/user.api';
+import { IUser } from '../../types/types';
+
+const Menu = () => {
+  const [userData, setUserData] = useState<IUser | null>(null);
+  const [selectedSection, setSelectedSection] = useState<string>('');
```

Comment:
We can make this better by creating a type that only allow certain values. Or the type can be based on the objects used to generate the list items. eg:
`{
sectionTitle: 'Profile'
icon: icon
alt: 'alt text'
}`